frontend/public/components/factory/table-filters.ts

```diff
@@ -22,7 +22,7 @@ export const fuzzyCaseInsensitive = (a: string, b: string): boolean =>
 
 // TODO: Table filters are undocumented, stringly-typed, and non-obvious. We can change that.
 export const tableFilters: TableFilterMap = {
-  name: (filter, obj) => fuzzyCaseInsensitive(filter, obj.metadata.name),
+  name: (filter, obj) => fuzzyCaseInsensitive(filter, _.get(obj, 'metadata.name', _.get(obj, 'name'))),
```

Comment:
Don't change the default name filter. We should add a new filter specifically for catalog sources.

Reply:
ok, fixed -thanks